lua/nvim-treesitter/locals.lua

```diff
@@ -127,17 +127,20 @@ end
 -- @param The full match path to append to
 -- @param The last match
 function M.recurse_local_nodes(local_def, accumulator, full_match, last_match)
-  if local_def.node then
-    accumulator(local_def, local_def.node, full_match, last_match)
-  else
-    for match_key, def in pairs(local_def) do
-      M.recurse_local_nodes(
-        def,
-        accumulator,
-        full_match and (full_match..'.'..match_key) or match_key,
-        match_key)
-    end
-  end
+	if(type(local_def) ~= "table") then return end
+
+	if local_def.node then
+		accumulator(local_def, local_def.node, full_match, last_match)
+	else
+		for match_key, def in pairs(local_def) do
+			M.recurse_local_nodes(
+			def,
+			accumulator,
+			full_match and (full_match..'.'..match_key) or match_key,
+			match_key)
```

Comment:
Let's keep the indentation of the arguments as how it was before. Think we should adopt one of those autoformatters in the future :)

Reply:
Yeah an autoformatter would be nice. The style checker doesn't seem to pick this up either. Anyway, should be fixed now

Reply:
We have an auto formatter now :)

```
cargo install stylua
stylua .
```

https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#style-checks-and-tests

Can you please rebase/merge master into this branch? There are conflicts.

Reply:
Sorry this small change is taking so long to be merged.

Reply:
I've tried resolving it now and running stylua to make it look nicer :)